.env

```diff
@@ -33,3 +33,5 @@ APP_LOG_SUBJECT="[Wikimedia OCR]"
 APP_IMAGE_HOSTS=upload.wikimedia.org,upload.wikimedia.beta.wmflabs.org
 
 APP_CACHE_TTL=3600 # 1 hour
+
+APP_TRANSKRIBUS_CLIENT_ID=
```

Comment:
Sorry I didn't ask this earlier, but why is this configurable? It appears to only have a single possible value, of `processing-api-client`. Is that right? Can this just be hardcoded?